bigquery/api/GettingStarted/runTests.ps1

```diff
@@ -1,17 +1,26 @@
-﻿# Copyright(c) 2016 Google Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not
-# use this file except in compliance with the License. You may obtain a copy of
-# the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations under
-# the License.
-Import-Module ..\..\..\BuildTools.psm1 -DisableNameChecking
-
-Build-Solution
-MSTest /testcontainer:test\bin\debug\test.dll
+﻿# Copyright(c) 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+Import-Module ..\..\..\BuildTools.psm1 -DisableNameChecking
+
+# Update Quickstart sample to include Project ID from Environment Variable
+$filePath = "QuickStart\Program.cs"
+$projectId = $env:GOOGLE_PROJECT_ID

```

Comment:
Add a TODO to move this code into BuiltTools.psm1.  Don't do it yet.  Let's write a couple more quick starts first so we figure out what code is actually common.